src/components/tour/tour_step_indicator.tsx

```diff
@@ -93,7 +82,16 @@ export const EuiTourStepIndicator: FunctionComponent<EuiTourStepIndicatorProps>
       values={{ status, number }}
     >
       {(ariaLabel: string) => (
-        <li className={classes} aria-label={ariaLabel} {...rest}>
+        <li
+          css={[
+            css`
+              display: inline-block;
+            `,
+          ]}
```

Comment:
Sorry, I think my previous comment was aa bit lost in translation. I didn't mine the `const styles` declaration, it's just the array I don't think is needed.
```suggestion
          css={css`
            display: inline-block;
          `}
```